src/js/components/search/filters/awardType/AwardType.jsx

```diff
@@ -10,40 +10,42 @@ import { awardTypeGroups, awardTypeCodes } from 'dataMapping/search/awardType';
 import PrimaryCheckboxType from 'components/sharedComponents/checkbox/PrimaryCheckboxType';
 import SubmitHint from 'components/sharedComponents/filterSidebar/SubmitHint';
 
-const defaultProps = {
-    awardTypes: [
-        {
-            id: 'award-contracts',
-            name: 'Contracts',
-            filters: awardTypeGroups.contracts
-        },
-        {
-            id: 'indefinite-delivery-vehicle',
-            name: 'Contract IDVs',
-            filters: awardTypeGroups.idvs
-        },
-        {
-            id: 'award-grants',
-            name: 'Grants',
-            filters: awardTypeGroups.grants
-        },
-        {
-            id: 'award-direct-payments',
-            name: 'Direct Payments',
-            filters: awardTypeGroups.direct_payments
-        },
-        {
-            id: 'award-loans',
-            name: 'Loans',
-            filters: awardTypeGroups.loans
-        },
-        {
-            id: 'award-other',
-            name: 'Other',
-            filters: awardTypeGroups.other
-        }
-    ]
-};
+const awardTypesData = [
+    {
+        id: 'award-contracts',
+        name: 'Contracts',
+        filters: awardTypeGroups.contracts
+    },
+    {
+        id: 'indefinite-delivery-vehicle',
+        name: 'Contract IDVs',
+        filters: awardTypeGroups.idvs
+    },
+    {
+        id: 'award-grants',
+        name: 'Grants',
+        filters: awardTypeGroups.grants
+    },
+    {
+        id: 'award-direct-payments',
+        name: 'Direct Payments',
+        filters: awardTypeGroups.direct_payments
+    },
+    {
+        id: 'award-loans',
+        name: 'Loans',
+        filters: awardTypeGroups.loans
+    },
+    {
+        id: 'award-other',
+        name: 'Other',
+        filters: awardTypeGroups.other
+    }
+];
+
+const awardTypeCodesData = Object.assign(awardTypeCodes, {
```

Comment:
You could also update the data mapping [here](https://github.com/fedspendingtransparency/usaspending-website/blob/DEV-2226_IDV_B/src/js/dataMapping/search/awardType.js#L25) instead of using `Object.assign`

Reply:
@ebdabbs ,

Are we sure that it's okay to update it there? Not sure if we want to remove this everywhere, or if removing it from here will remove it everywhere in the app...?

Reply:
@MaxwellKendall yep, in this case we do want to update it everywhere since without `Calls` is the more correct terminology for IDVs.

Reply:
Alright!!